KataTest/StringCalculatorTestCase.cs

```diff
@@ -0,0 +1,72 @@
+using KataStringCalculator;
+using Xunit;
+namespace KataTest
+{
+    public class StringCalculatorTestCase
+    {
+        [Fact]
```

Comment:
Use theory & member data with tests that shared the same behavior with different data